src/index.html

```diff
@@ -11,7 +11,129 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset>
+
+        <legend>Personal information</legend>
+
+        <label class="block">
+          Surname:
+          <input type="text" required
+           autocomplete="off"
+          >
+        </label>
+
+        <label class="block">
+          Name:
+          <input type="text" autocomplete="off">
+        </label>
+         
+        <label class="block">
+          How old are You? 
+          <input type="number" value="12"
+           min="1" max="100"
+          >
```

Comment:
Pay attention on indents. Fix all places
```suggestion
          <input type="number" value="12"
            min="1" max="100"
          >
```